app/base.scss

```diff
@@ -0,0 +1,203 @@
+$brand-black: #000;
+$brand-white: #fff;
+$brand-tiff: #1ef3ed;
+$brand-grey: #a7a7a7;
+$brand-lime: #1ef383;
+$pad: 1em;
+
+@mixin keyframes($name) {
+  @-webkit-keyframes #{$name} {
+    @content
+  }
+  @-moz-keyframes #{$name} {
+    @content
+  }
+  @-ms-keyframes #{$name} {
+    @content
+  }
+  @-o-keyframes #{$name} {
+    @content
+  }
+  @keyframes #{$name} {
+    @content
+  }
+}
+
+@mixin animation($value) {
+  -webkit-animation: $value;
+  -moz-animation: $value;
+  -ms-animation: $value;
+  -o-animation: $value;
+  animation: $value;
+}
+
+@include keyframes(move) {
+  0% { left: 0; }
+  100% { left: 100px; }
+}
+
+.box {
```

Comment:
Totally cool with you copy/pasting sample ``scss`` stuff you find on the Internet, but make sure you try to understand what things are doing before copying/pasting stuff that seems cool.